apollo.py

```diff
@@ -43,6 +43,7 @@
     "cogs.commands.widen",
     "cogs.commands.xkcd",
     "cogs.channel_checker",
+    "cogs.commands.system",
```

Comment:
not necessary but i thought we were going on alphabetical ordering for cogs?

Reply:
no idea but will change

Reply:
In general yea, think it's been broken a couple times, but that's the general idea